Guidelines for scientific documentation (#77)
CABLE-LSM/CABLE: documentation/docs/developer_guide/git_process.md

# Process to develop and submit new features

This process is the same for any submission to be made to CABLE:

 - a bug fix
 - a new feature for CABLE with its documentation
 - a change in the documentation only
 - etc.

> Since the source code and the documentation are held in text files under the same repository, there are no distinction made about updating one or the other. 

Reviewer: ```suggestion
Since the source code and the documentation are held in text files in the same repository, the process for updating them is the same. 
```

> Developers should submit updates to the documentation at the same time they submit updates to the source code as required. Submission with changes to the source code that do not contain the required updates to the documentation will not be accepted for release, the author will be asked to add the modifications to the documentation before acceptance.

Reviewer: ```suggestion
Developers should submit updates to the documentation when they submit updates to the source code. Submissions with changes to the source code that do not contain appropriate documentation updates will not be accepted, and the author will be asked to add documentation.
```
